scripts/who/gho/test_data/who.csv

```diff
@@ -0,0 +1,434 @@
+year,period,country,statVar,value
+2015,P1Y,country/ARG,dcs:EMFLIMITELECTRIC_Public_LowFrequency,3
+2014,P1Y,country/AUS,dcs:EMFLIMITELECTRIC_Public_LowFrequency,'[5]/[10]'

```

Comment:
EMFLIMITELECTRIC_Public_LowFrequency in the data has "null" for the numericValue field. Right now we interpret statvars where the numericValue field is null as measurementResult because it's not guaranteed that the value in the value field will be numeric..